### PR TITLE
Add company model 

### PR DESCRIPTION
**Pull request by quyenlq**
## Description :sparkles:
First change to add Company model to the project
The `Company` is inherited using `shared` model `AbstractCompany` and added `IBANField` for our specific needs
API will come later in a separated PR

### CODE REVIEW

File: backend/benefit/companies/admin.py [hunk 1]
```diff
@@ -0,0 +1,3 @@
+from django.contrib import admin  # noqa
+
+# Register your models here.
```
Comment (haavikko) [inline backend/benefit/companies/admin.py:3]: maybe register model / add basic ModelAdmin?

File: .github/workflows/bf-pytest.yml [hunk 2]
```diff
@@ -49,7 +49,7 @@ jobs:
             ${{ runner.os }}-
 
       - name: Install dependencies
-        run: cd backend/benefit && pip install -r requirements.txt -r requirements-dev.txt
+        run: cd backend/benefit && pip install -r requirements.txt -r requirements-dev.txt codecov
```
Comment (haavikko) [inline .github/workflows/bf-pytest.yml:52]: +1, I noticed the same in github

File: backend/benefit/common/tests/conftest.py [hunk 3]
```diff
@@ -0,0 +1,10 @@
+import factory.random
+import pytest
+from django.test.utils import freeze_time
+
+
+@pytest.fixture(autouse=True)
+def setup_test_environment(settings):
+    factory.random.reseed_random("777")
+    with freeze_time("2020-01-04"):
```
Comment (haavikko) [inline backend/benefit/common/tests/conftest.py:9]: 2021 instead? Some business rules might depend on date?

File: backend/benefit/companies/apps.py [hunk 4]
```diff
@@ -0,0 +1,6 @@
+from django.apps import AppConfig
+
+
+class CompanyConfig(AppConfig):
+    default_auto_field = "django.db.models.BigAutoField"
```
Comment (haavikko) [inline backend/benefit/companies/apps.py:5]: +1

File: backend/benefit/companies/models.py [hunk 5]
```diff
@@ -0,0 +1,13 @@
+from django.utils.translation import gettext_lazy as _
+from localflavor.generic.models import IBANField
+
+from shared.models.abstract_models import AbstractCompany
+
+
+class Company(AbstractCompany):
+    bank_account_number = IBANField(
+        include_countries=("FI",), verbose_name=_("bank account number")
+    )
```
Comment (haavikko) [inline backend/benefit/companies/models.py:10]: Are all fields here that we discussed in the sprint planning?

Comment (quyenlq) [reply backend/benefit/companies/models.py:10]: Yeah I think that pretty much all we need. The rest of the fields are inherited from the AbstractCompany fields

File: backend/benefit/companies/admin.py (hunk 1 above)
Comment (quyenlq) [reply backend/benefit/companies/admin.py]: Yeah, totally didn't look into this file :D Fixed now

File: backend/benefit/common/tests/conftest.py (hunk 3 above)
Comment (quyenlq) [reply backend/benefit/common/tests/conftest.py]: Fixed now 👌

File: backend/shared/shared/models/abstract_models.py [hunk 6]
```diff
@@ -32,9 +32,13 @@ class Meta:
 
 
 class AbstractCompany(models.Model):
```
Comment (klemmari1) [inline backend/shared/shared/models/abstract_models.py:34]: Should this inherit UUIDModel?

**Review by klemmari1 — APPROVED** [review]
Good stuff!

**Pull request merged**